lib/cache/cache.go

```diff
@@ -901,6 +914,13 @@ func (c *Cache) fetchAndWatch(ctx context.Context, retry utils.Retry, timer *tim
 
 	retry.Reset()
 
+	relativeExpiryInterval := interval.New(interval.Config{
+		Duration:      c.Config.RelativeExpiryCheckInterval,
+		FirstDuration: utils.HalfJitter(c.Config.RelativeExpiryCheckInterval),
```

Comment:
Unfortunately, yes.  That would be the case even without jitter, and is somewhat true today anyway (poor auth server performance or backend API client throttling can lead to cache's being out of sync in time).

The best mitigation I could come up with was to have the auth server emit its expiry events to downstream caches rather than having downstream caches expire entries independently.  This limits the number of competing views considerably.  It also means that we'll be able to easily trace any bugs, since auth caches can be queried by `tctl`, but proxy caches cannot.

Disagreement could be further minimized by reducing the `RelativeCheckExpiryInterval` if need be.